### PR TITLE
WIP: release/2.1.0

### PR DESCRIPTION
**Pull request by asitnkova**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,8 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Compilation error if View interface method's parameter name clash with generated code
 
 ### Added
-- `MvpPresenter.presenterScope` extension for kotlin coroutines
+- Added `MvpPresenter.presenterScope` extension for kotlin coroutines
 - Added ability to define alias annotations for `@StateStrategyType` annotation, and added some default aliases to the library. More info [in this issue](https://github.com/moxy-community/Moxy/issues/72)
+- Added `@Inherited` annotation to `@InjectViewState`. It means `@InjectViewState` is now optional, `ViewState` will be generated even without explicit annotation on `MvpPresenter` subclass
```
Comment (jordan1997) [inline CHANGELOG.md]: May be, we can mark `@InjectViewState` as Depracted

File: README.md [hunk 2]
```diff
@@ -49,26 +39,33 @@ Presenter
 @InjectViewState
```
Comment (jordan1997) [inline README.md]: We can remove @InjectVIewState from the sample project.

File: CHANGELOG.md (hunk 1 above)
Comment (asitnkova) [reply CHANGELOG.md]: Well, it's not technically deprecated, and I think it will be more annoying than helpful

Comment (asitnkova) [reply CHANGELOG.md]: It cannot be done unless we stop generating code. Annotation is still needed for annotation processing to start

File: README.md (hunk 2 above)
Comment (asitnkova) [reply README.md]: Done

**Pull request merged**